### DISCUSSION TITLE
Nice work but ....:)

### DISCUSSION THREAD

**Discussion by Product-UX** [heroui-inc/heroui#208] [Feedback]
Hi
I am a UX designer and I have shared your work with my Dev team but I would like to ask you to please fix this contrast issue on the "danger" Buttons or Chips. This porky pink does not work with electric pink not contrast and actually hunting the eyes. Cheers 

<img width="688" alt="Screenshot 2022-01-31 at 17 58 07" src="https://user-images.githubusercontent.com/91541841/151838209-2c4b9581-858e-4b0a-927a-8ba7235dfb2d.png">

**Comment by jrgarciadev** [comment]
Hey @Product-UX thanks for the feedback we will fix that 👍🏻
